docs/references.bib

```diff
@@ -293,3 +293,120 @@ @article{spanjaard:18
+  title     = {Lineage tracing meets single-cell omics: opportunities and challenges},
+  volume    = {21},
+  year      = {2020},
+}
+
+@article{jones:20,
+  author    = {Jones, Matthew G and Khodaverdian, Alex and Quinn, Jeffrey J and Chan, Michelle M and Hussmann, Jeffrey A and Wang, Robert and Xu, Chenling and Weissman, Jonathan S and Yosef, Nir},
+  publisher = {BioMed Central},
+  journal   = {Genome biology},
+  number    = {1},
+  pages     = {1--27},
+  title     = {Inference of single-cell phylogenies from lineage tracing data using Cassiopeia},
+  volume    = {21},
+  year      = {2020},
+}
+
+@article{forrow:21,
+  author    = {Forrow, Aden and Schiebinger, Geoffrey},
+  publisher = {Nature Publishing Group UK London},
+  journal   = {Nature communications},
+  number    = {1},
+  pages     = {4940},
+  title     = {LineageOT is a unified framework for lineage tracing and trajectory inference},
+  volume    = {12},
+  year      = {2021},
+}
+
+@article{raj:18,
+  author    = {Raj, Bushra and Wagner, Daniel E and McKenna, Aaron and Pandey, Shristi and Klein, Allon M and Shendure, Jay and Gagnon, James A and Schier, Alexander F},
+  publisher = {Nature Publishing Group US New York},
+  journal   = {Nature biotechnology},
+  number    = {5},
+  pages     = {442--450},
+  title     = {Simultaneous single-cell profiling of lineages and cell types in the vertebrate brain},
+  volume    = {36},
+  year      = {2018},
+}
+
+@article{alemany:18,
+  author    = {Alemany, Anna and Florescu, Maria and Baron, Chloé S and Peterson-Maduro, Josi and Van Oudenaarden, Alexander},
+  publisher = {Nature Publishing Group UK London},
+  journal   = {Nature},
+  number    = {7699},
+  pages     = {108--112},
+  title     = {Whole-organism clone tracing using single-cell sequencing},
+  volume    = {556},
+  year      = {2018},
+}
+
+@article{peyre:19,
+  author    = {Peyré, Gabriel and Cuturi, Marco and others},
+  publisher = {Now Publishers, Inc.},
+  journal   = {Foundations and Trends{®} in Machine Learning},
+  number    = {5-6},
+  pages     = {355--607},
+  title     = {Computational optimal transport: With applications to data science},
+  volume    = {11},
+  year      = {2019},
+}
+
+@article{chizat:18,
+  author  = {Chizat, Lenaic and Peyré, Gabriel and Schmitzer, Bernhard and Vialard, François-Xavier},
+  journal = {Mathematics of Computation},
+  number  = {314},
+  pages   = {2563--2609},
+  title   = {Scaling algorithms for unbalanced optimal transport problems},
+  volume  = {87},
+  year    = {2018},
+}
+
+@article{lange-moslin:23,
+  author    = {Lange, Marius and Piran, Zoe and Klein, Michal and Spanjaard, Bastiaan and Klein, Dominik and Junker, Jan Philipp and Theis, Fabian J. and Nitzan, Mor},
+  publisher = {Cold Spring Harbor Laboratory},
+  url       = {TODO},
+  doi       = {TODO},
+  journal   = {bioRxiv},
+  title     = {Mapping lineage-traced cells across time points with moslin},
+  year      = {2023},
+}
```

Comment:
to do: add scot reference? How can I avoid merging conflict?

Reply:
what merge conflict do you have?

Reply:
also, please run `tox -e format-references`